app/src/screens/SelectNFTSection.tsx

```diff
@@ -70,12 +70,23 @@ export default function SelectNFTSection() {
 
           <ENSDisplay />
 
-          {name && (
-            <View style={[styles.spaced, isMoWeb && styles.spacedXS]}>
-              <Typography>Select NFT or upload image</Typography>
-              <NFTSelectorCloudPure selectedIndex={nftIndex} onSelect={setNft} />
-            </View>
-          )}
+          <View style={[styles.spaced, isMoWeb && styles.spacedXS]}>
+            {loadingNfts && <ActivityIndicator size={24} />}
+            {openseaError && <Typography>Unable to search wallet for NFTs. Please try again later.</Typography>}
```

Comment:
Update this to reference that error:
`{error && <Typography>{error} Please try again later.</Typography>}`